app/src/main/java/org/wikipedia/suggestededits/PageSummaryForEdit.kt

```diff
@@ -12,6 +12,7 @@ data class PageSummaryForEdit(
         var displayTitle: String?,
         var description: String?,
         var thumbnailUrl: String?,
+        var extract: String? = null,
```

Comment:
What's the goal of this new `extract` parameter, when we already have an `extractHtml` parameter below?

Since you're adding a new parameter in the middle of the parameter list, you must be careful to update all the places where this constructor is called.  Because of this, the main Add/Translate Descriptions workflow in Suggested Edits is currently broken.